packages/@aws-cdk/aws-codeartifact/test/validation.test.ts

```diff
@@ -0,0 +1,35 @@
+import '@aws-cdk/assert/jest';
+import { validate } from '../lib/validation';
+
+test('Is undefined and value is empty', () => {
+  validate('test', {}, undefined);
+});
+
+test('Is Required and value is empty', () => {
+  testValidate('Is required and cannot be empty', { required: true }, undefined);
```

Comment:
Much more idiomatic:

```suggestion
  expect(() => validate('test', { required: true }, undefined)).toThrow(/Is required and cannot be empty/);
```